src/ImageGallery.jsx

```diff
@@ -459,17 +441,21 @@ export default class ImageGallery extends React.Component {
 
   _handleKeyDown = (event) => {
     const LEFT_ARROW = 37;
+    const H_KEY = 72;
     const RIGHT_ARROW = 39;
+    const L_KEY = 76;
     const ESC_KEY = 27;
```

Comment:
What makes H and L keys special here? I'm not sure why this is important.

Reply:
In unix editor vi you can move the cursor with l to right and h to left. So users from unix would like to use this feature.

Reply:
I want to avoid adding more key listeners to the library except the really obvious ones. For this I recommend using `slideToIndex` function and writing your own function to slide the gallery.

Reply:
That's ok for me.

Reply:
btw: even Github implemented the k and j keys from vi to move down or up. You can see it, if you open the Keyboard Shortcuts on the Github page by pressing `?`

Reply:
Probably makes sense since coders use github. Depending on who uses the library I want to interfere with user keyboard as little as possible.